pkgs/tools/networking/telepresence/default.nix

```diff
@@ -22,13 +22,13 @@ let
       });
 in pythonPackages.buildPythonPackage rec {
   pname = "telepresence";
-  version = "0.105";
+  version = "0.108";
 
   src = fetchFromGitHub {
     owner = "telepresenceio";
     repo = "telepresence";
     rev = version;
-    sha256 = "0fccbd54ryd9rcbhfh5lx8qcc3kx3k9jads918rwnzwllqzjf7sg";
+    sha256 = "1bidsmxgw4dyx1pb1kxawpza78lw5f3g4l0ym0sqxlvwg6kjqi4f";
```

Comment:
I get a hash mismatch here.

Was the release artifact modified in the meantime?

```
hash mismatch in fixed-output derivation '/nix/store/dchkph5d9iiivys61mjpwz9fzhfgj8rg-source':
  wanted: sha256:1bidsmxgw4dyx1pb1kxawpza78lw5f3g4l0ym0sqxlvwg6kjqi4f
  got:    sha256:1f3bamcsdgyjjd9k6ykqx7c2k9bm9qr26c5xh11i7nvy8qrjqpg9
cannot build derivation '/nix/store/q0774w45nggjqfa4jpwn7k767mb2lnmq-python3.8-telepresence-0.108.drv': 1 dependencies couldn't be built
```

Reply:
Seems to happen here too. Doesn't look like anything weird has been happening, the tag is dated to 2020-09-10, weeks before this PR. Maybe @benlivingood-wf had a previous patch that he updated, but forgot to change the hash?

Reply:
@benlivingood-wf